homeassistant/components/nina/config_flow.py

```diff
@@ -149,6 +150,7 @@ async def async_step_user(
                         int, vol.Range(min=1, max=20)
                     ),
                     vol.Optional(CONF_HEADLINE_FILTER, default=""): cv.string,
+                    vol.Optional(CONF_AREA_FILTER, default=""): cv.string,
```

Comment:
Looks maybe like this should be in options instead of data so you should implement and OptionsFlow?

Reply:
That is a good point, I did not considered that. It is better as an option.